Sources/Extensions/Foundation/Dictionary+Povio.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Dictionary+Povio.swift
+//  PovioKit
+//
+//  Created by Borut Tomažin on 11/11/2020.
+//  Copyright © 2020 Povio Labs. All rights reserved.
+//
+
+import Foundation
+
+public extension Dictionary {
+  /// Make a union on two dictionaries.
+  /// ```
+  /// let dictionaryA = [1: "One", 2: "Two"]
+  /// let dictionaryB = [3: "Three"]
+  /// let unionDictionary = dictionaryA.union(dictionaryB)
+  /// print(unionDictionary) // [1: "One", 2: "Two", 3: "Three"]
+  /// ```
+  func union(_ dictionary: Dictionary) -> Dictionary {
```

Comment:
What happens if dictionaries contain the same key. Which value remains?

Reply:
Original is preserved. Which is not good. I'll think about it.

Reply:
Actually, this is redundant. [Dictionary](https://developer.apple.com/documentation/swift/dictionary/3127169-merge) already has `merge` and `merging` methods.